app/(client)/event-calendar/page.tsx

```diff
@@ -0,0 +1,208 @@
+'use client';
+
+import { useState } from 'react';
+import { MapPinned, SortAsc, SortDesc } from 'lucide-react';
+
+import { Button } from '@/components/ui/button';
+import {
+  Command,
+  CommandEmpty,
+  CommandInput,
+  CommandItem,
+  CommandList,
+} from '@/components/ui/command';
+import {
+  Select,
+  SelectContent,
+  SelectItem,
+  SelectTrigger,
+} from '@/components/ui/select';
+
+const EventsList = [
+  {
+    value: 'first-year-day',
+    label: 'День перваша',
+  },
+  {
+    value: 'faculty-day',
+    label: 'День факультету',
+  },
+];
+
+const SearchForEvent = () => {
+  const [value, setValue] = useState('');
+  const [hasSetValue, setHasSetValue] = useState(false);
+
+  return (
+    <Command>
+      <CommandInput
+        placeholder="Шукати захід"
+        autoComplete="true"
+        value={value}
+        onValueChange={newValue => setValue(newValue)}
+        onClick={() => {
+          setHasSetValue(false);
+        }}
+      />
+      <CommandList className={hasSetValue ? 'hidden w-0 h-0' : 'block'}>
+        <CommandEmpty>Не вдалось знайти захід.</CommandEmpty>
+        {EventsList.map(i => (
```

Comment:
Searching must be through real data

Reply:
There is no real data yet. EventsList contains the data that should be gathered from API.